newscoop/classes/NodeTree.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
Because this class is tightly coupled with the Node class which is already in the classes folder, I think it makes sense for it to be there, no?